### PR TITLE
[13.0] [MIG] project_custom_info: MIgration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26771

please @Tardo @carlosdauden review this

### CODE REVIEW

**Review by Tardo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Can you please take the occasion to add demo data to this module? You can take as inspiration demo data in base_custom_info.

**Comment by CarlosRoca13** [comment]
@pedrobaeza demo data added

**Review by lfreeke — APPROVED** [review]
LGTM :+1: (Functional test)

File: project_custom_info/demo/custom.info.template.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model,model_id:id
+tpl_addressed,Addressed to...,res.partner,base.model_res_partner
```
Comment (pedrobaeza) [inline project_custom_info/demo/custom.info.template.csv]: This is not correct, as the model should be project. And the idea is to put some data related to project, not PEGI.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
I changed the demo data, can you review it again @pedrobaeza

File: project_custom_info/demo/default.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Tecnativa - Carlos Roca
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<odoo>
+    <record id="prop_time" model="custom.info.property">
+        <field name="default_value">3</field>
```
Comment (pedrobaeza) [inline project_custom_info/demo/default.xml]: Why not including this on the CSV?

File: project_custom_info/demo/project_project.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Tecnativa - Carlos Roca
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<odoo>
+    <record id="project_with_custom_info_demo" model="project.project">
+        <field name="name">Create a module for allow custom info on models</field>
```
Comment (pedrobaeza) [inline project_custom_info/demo/project_project.xml]: After `for`, a gerund is always needed (`for allowing`), but let's shorten it:
```suggestion
        <field name="name">Module analysis</field>
```

File: project_custom_info/demo/default.xml (hunk 2 above)
Comment (CarlosRoca13) [reply project_custom_info/demo/default.xml]: I'm based on base_custom_info. In these module, the default value was added like here... I don't know the reason but I thought that's important.

Comment (pedrobaeza) [reply project_custom_info/demo/default.xml]: Please try the other way to see. I don't find right now the reason for this.

**Comment by CarlosRoca13** [comment]
Changes done @pedrobaeza the default value on csv is working fine

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**